source/octf/trace/iotrace_event.h

```diff
@@ -37,10 +37,13 @@ typedef enum {
     iotrace_event_type_io_cmpl = 'C',
 
     /** Filesystem meta information event */
-    iotrace_event_type_fs_meta = 'F',
+    iotrace_event_type_fs_meta = 'M',
```

Comment:
I think we can remove custom numerating. Please remember about updating trace version above.